src/main/kotlin/frc/robot/ControllerCommands.kt

```diff
@@ -0,0 +1,22 @@
+package frc.robot
+
+import edu.wpi.first.wpilibj.GenericHID
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.Commands
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController
+
+fun startRumble(controller: CommandXboxController): Command {
+    return Commands.runOnce(
+        {
+            controller.hid.setRumble(GenericHID.RumbleType.kBothRumble, 1.0)
+        }
+    )
+}
+
+fun stopRumble(controller: CommandXboxController): Command {
+    return Commands.runOnce(
+        {
+            controller.hid.setRumble(GenericHID.RumbleType.kBothRumble, 0.0)
+        }
+    )
```

Comment:
Code duplication

Reply:
3142e87a076f8b48df41900f38e673d1c1e773a7

Reply:
First rumble should be private

Reply:
Oops 126b9140c07b7aa1d845bdfafd577ade1d8c7ecd